src/index.js

```diff
@@ -0,0 +1,195 @@
+"use strict";
+
+const MAX_DEGREE = 60;
+const MIN_DEGREE = -90;
+
+const state ={
+  //HTML states
+  increaseTempControl: null,
+  tempValue: null,
+  decreaseTempControl:null,
+  landscape: null,
+  headerCityName: null,
+  cityNameInput: null,
+  cityNameReset: null,
+  currentTempButton: null,
+  skySelect: null,
+  sky__section: null,
+  sky: null,
+  gardenContent: null,
+
+  tempValueCount: 0,
+};
+
+const loadControls = () => {
+  state.increaseTempControl = document.getElementById("increaseTempControl");
+  state.tempValue = document.getElementById("tempValue");
+  state.decreaseTempControl = document.getElementById("decreaseTempControl");
+  state.landscape = document.getElementById("landscape");
+  state.headerCityName = document.getElementById("headerCityName");
+  state.cityNameInput = document.getElementById("cityNameInput");
+  state.cityNameReset = document.getElementById("cityNameReset");
+  state.currentTempButton = document.getElementById("currentTempButton");
+  state.skySelect = document.getElementById("skySelect")
+  state.sky__section = document.getElementsByClassName("sky__section")
+  state.sky = document.getElementsByClassName("sky")
+  state.gardenContent = document.getElementById('gardenContent');
+};
+
+const getCurrentTemperature = () => {
+  const city = state.cityNameInput.value;
+
+  return axios
+    .get('http://localhost:5000/location', {params: {
+      q: city,
+      },
+    })
+    .then((response) => {
+      const locationData = response.data[0];
+      const cityName = locationData.city;
+
+      state.headerCityName.textContent = cityName;
+      handleCityName();
+      axios.get(`http://localhost:5000/weather?lat=${locationData.lat}&lon=${locationData.lon}`)
+      .then((response)=>{
+          const temperature = response.data.main.temp;
+          const tempCelsius = Math.floor(temperature - 273.15);
+          state.tempValueCount = tempCelsius;
+          state.tempValue.textContent = tempCelsius;
+          console.log(response.data)
+          handleChangeBackgroundColor();
+      })
+      .catch((error) => {
+        console.log("Error temperature:", error);
+      });
+    })
+    .catch((error) =>{
+      console.log("Error location:", error);
+    });
+  };
+
+const skySelection = () => {
+  const skySelect = state.skySelect.value;
+
+  switch (skySelect) {
+    case 'Sunny':
+      sky.textContent = "☀️ ☀️ ☀️ ☀️ ☀️";
+      gardenContent.classList = "garden__content sunny";
```

Comment:
Consider building a dictionary where the key is the sky setting, and the values hold the emoji string and display class. Then rather than writing a `switch`, we could look up the values directly in the dictionary, simplifying the code and making it more data driven. We could even enumerate the keys and use them to dynamically build the options in the sky selection!